consensus/consensus.go

```diff
@@ -185,3 +188,9 @@ type ConsensusInfo struct {
 	Committers     []common.Address
 	Round          byte
 }
+
+type ChainContext interface {
```

Comment:
You don't need separate ChainContext type. ChainReader already has these.

Reply:
There are three callsites in api package. `api.backend` does not satisfy the interface of `ChianReader`, so I must create a minimal interface type.

Reply:
How about adding missing methods here instead?

https://github.com/kaiachain/kaia/blob/8fb6105b4c0455caefc67caf7b314b40b7333912/blockchain/evm.go#L42-L50

Reply:
Uhm,, really? The `Backend` seems too bloated to match the defined function names. I believe that's why the `ChainContext` type exists. I'm okay with extending one, but the impact of changes seems to be considered before making a change.

Reply:
I confirmed that tracers/api.go:backend does not satisfy ChainReader and there is a wrapper class chainContext specifically for calling NewEVMBlockContext. I think having consensus.ChainContext is our best bet in the current situation. After this PR, let me propose another PR that simplifies tracer.